modules/UI/videolayout/Filmstrip.js

```diff
@@ -55,7 +55,14 @@ const Filmstrip = {
             width: `${avatarSize}px`
         });
 
-        [ ...thumbs.remoteThumbs, ...thumbs.localThumb ].forEach(videoThumb => {
+        const thumbEls = [ ...thumbs.remoteThumbs ];
+
+        // localThumb is underfined when iAmRecorder is enabled
+        if (thumbs.localThumb) {
+            thumbEls.concat([ ...thumbs.localThumb ]);
+        }
+
+        thumbEls.forEach(videoThumb => {
```

Comment:
```suggestion
       thumbs.localThumb = thumbs.localThumb || [];
       [ ...thumbs.remoteThumbs, ...thumbs.localThumb ].forEach(videoThumb => {
```

Reply:
I don't want to change the reference, who knows how it that used

Reply:
`[ ...thumbs.remoteThumbs, ...thumbs.localThumb || [] ]`